sed/test/inspec/controls/command_exists.rb

```diff
@@ -0,0 +1,8 @@
+pkg_ident = `grep pkg_ident results/last_build.env | cut -d \= -f2`.strip
+executable_path = "/hab/pkgs/#{pkg_ident}/bin/sed"
+
+control 'The sed exe exists in the last built package' do
```

Comment:
These tests are intentionally prickly about only testing the executable from `results/last_build.env`
They won't work if another version of the pkg is installed. 
This should be expressed well to people examining this repo - I'll at least make this clear in the README

Reply:
Added to README and also added `control` descriptions to the tests so they are self-explanatory.